stable/sumologic-fluentd/Chart.yaml

```diff
@@ -1,5 +1,5 @@
 name: sumologic-fluentd
-version: 0.12.0
+version: 0.12.1
```

Comment:
Shouldn't this be 0.13?

Reply:
hrmm, I guess it should since it's new, let me amend that.